packages/components/src/overlays/Popover/src/Popover.module.css

```diff
@@ -0,0 +1,96 @@
+:where(.hop-Popover *) {
+    font-family: var(--font-family);
+    font-size: var(--font-size);
+    font-weight: var(--font-weight);
+    line-height: var(--line-height);
+    color: var(--color);
+
+    outline: none;
+}
+
+.hop-Popover__dialog {
+    --hop-Popover-background: var(--hop-neutral-surface);
+    --hop-Popover-font-family: var(--hop-body-sm-font-family);
+    --hop-Popover-font-size: var(--hop-body-sm-font-size);
+    --hop-Popover-font-weight: var(--hop-body-sm-font-weight);
+    --hop-Popover-line-height: var(--hop-body-sm-line-height);
+    --hop-Popover-color: var(--hop-neutral-text);
+    --hop-Popover-title-font-family: var(--hop-body-sm-font-family);
+    --hop-Popover-title-font-weight: var(--hop-body-sm-semibold-font-weight);
+    --hop-Popover-content-font-size: var(--hop-body-sm-font-size);
+    --hop-Popover-content-line-height: var(--hop-body-sm-line-height);
+    --hop-Popover-content-color: var(--hop-neutral-text-weak);
+    --hop-Popover-footer-font-size: var(--hop-body-sm-font-size);
+    --hop-Popover-footer-line-height: var(--hop-body-sm-line-height);
+    --hop-Popover-footer-link-color: var(--hop-primary-text);
+    --hop-Popover-border: 0.0625rem solid var(--hop-neutral-border-weak);
+    --hop-Popover-border-radius: var(--hop-shape-rounded-md);
+    --hop-Popover-box-shadow: var(--hop-elevation-lifted);
+    --hop-Popover-max-inline-width: calc(100% - 1rem);
+    
+    /* Internal variables */
+    --font-family: var(--hop-Popover-font-family);
+    --font-size: var(--hop-Popover-font-size);
+    --font-weight: var(--hop-Popover-font-weight);
+    --line-height: var(--hop-Popover-line-height);
+
+    display: grid;
+    grid-template-areas:
+        "header header"
+        "content content"
+        "footer action";
+    grid-template-columns: repeat(2, auto);
+    gap: var(--hop-space-inset-sm);
+
+    box-sizing: border-box;
```

Comment:
this should be on every element

Reply:
Do you mean in a general way in the components?

Reply:
for example, if you look at Switch css, it's on the switch component and each sub element that isn't already a hopper component.

Reply:
In my case, it doesn't seem necessary to use it on every sub-element. The purpose of this declaration is to include or exclude padding on an element to which a size is defined. In Popover, none of the children have a defined size (width or height).